core/pom.xml

```diff
@@ -0,0 +1,421 @@
+        <trilead-ssh2.version>build-217-jenkins-16</trilead-ssh2.version>
+        <opentelemetry.version>1.22.0</opentelemetry.version>
+        <opentelemetry.alpha.version>1.22.0-alpha</opentelemetry.alpha.version>
+        <testng.version>7.4.0</testng.version>
+        <bouncycastle-jdk15on.version>1.64</bouncycastle-jdk15on.version>
+        <jsftemplating.version>3.0.0</jsftemplating.version>
+        <jakarta.faces-api.version>4.0.1</jakarta.faces-api.version>
+
+        <!-- build versions -->
+        <jdk.version>11</jdk.version>
+        <maven.enforcer.plugin.version>3.1.0</maven.enforcer.plugin.version>
+        <command-security-plugin.version>1.0.13</command-security-plugin.version>
+        <command.security.maven.plugin.isFailureFatal>false</command.security.maven.plugin.isFailureFatal>
+        <maven.jar.plugin.version>3.3.0</maven.jar.plugin.version>
+        <maven.clean.plugin.version>3.2.0</maven.clean.plugin.version>
+        <maven.compiler.plugin.version>3.10.1</maven.compiler.plugin.version>
+        <maven.compiler.showDeprecation>true</maven.compiler.showDeprecation>
+        <maven.surefire.plugin.version>3.0.0-M7</maven.surefire.plugin.version>
+        <maven.resources.plugin.version>3.3.0</maven.resources.plugin.version>
+        <maven.release.plugin.version>2.5.3</maven.release.plugin.version>
+        <maven.dependency.plugin.version>3.3.0</maven.dependency.plugin.version>
+        <maven.gpg.plugin.version>1.6</maven.gpg.plugin.version>
+        <maven.deploy.plugin.version>3.0.0</maven.deploy.plugin.version>
+        <maven.source.plugin.version>3.2.1</maven.source.plugin.version>
+        <maven.javadoc.plugin.version>3.4.1</maven.javadoc.plugin.version>
+        <maven.install.plugin.version>3.1.0</maven.install.plugin.version>
+        <maven.bundle.plugin.version>4.1.0</maven.bundle.plugin.version>
+        <maven.failsafe.plugin.version>3.0.0-M7</maven.failsafe.plugin.version>
+        <maven.site.plugin.version>3.12.1</maven.site.plugin.version>
+        <glassfishbuild.version>3.2.20.payara-p2</glassfishbuild.version>
+        <maven.antrun.plugin.version>3.1.0</maven.antrun.plugin.version>
+        <maven.assembly.plugin.version>3.4.2</maven.assembly.plugin.version>
+        <hk2.plugin.version>3.0.2</hk2.plugin.version>
+        <maven.build.helper.plugin.version>3.3.0</maven.build.helper.plugin.version>
+
+
+        <!-- build settings -->
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <deploy.skip>true</deploy.skip>
+        <javadoc.skip>true</javadoc.skip>
+    </properties>
+
+    <scm>
+        <connection>scm:git:git@github.com:payara/payara.git</connection>
+        <url>scm:git:git@github.com:payara/payara.git</url>
+        <developerConnection>scm:git:git@github.com:payara/payara.git</developerConnection>
+        <tag>payara-server-6.2023.3-SNAPSHOT</tag>
```

Comment:
This is now wrong.
Also, I feel like the core should probably have its own tag?

Something along the lines of `payara-core-6.x.y`?